networkx/algorithms/__init__.py

```diff
@@ -78,6 +78,7 @@
 from networkx.algorithms import operators
 from networkx.algorithms import shortest_paths
 from networkx.algorithms import tournament
+from networkx.algorithms.tournament import is_tournament
```

Comment:
This need to go down where other functions are imported. Look at line 85 and below in the same file.

Reply:
done!